### DISCUSSION TITLE
How to display HTML

### DISCUSSION THREAD

**Discussion by AleidaBOX** [cefsharp/CefSharp#3457] [Q&A] [answered]
I've been wanting to replace the Blank screen when handing load errors by using the ILoadErrorHandler class but I don't know how to correctly display HTML in CefSharp 88.2.90 I've been needing some assistance 

![](https://cloud.githubusercontent.com/assets/456638/6598206/183efee0-c7d0-11e4-9ebb-bc1490bdb350.png)

![](https://cloud.githubusercontent.com/assets/456638/6598204/16551164-c7d0-11e4-869a-f908407d6ccd.png)
 
 Simply I wanna create something like this if anyone can guide me on how to properly display HTML in 88.2.90 then Thanks.

**Comment by amaitland** [comment, marked as answer]
> handing load errors by using the ILoadErrorHandler

There's no `ILoadErrorHandler` interface, possibly you mean [ILoadHandler.OnLoadError](http://cefsharp.github.io/api/88.2.x/html/M_CefSharp_ILoadHandler_OnLoadError.htm)

See the following for an example using the `LoadError` event, you can use `ILoadHandler.OnLoadError` if you choose, they're the same.
https://github.com/cefsharp/CefSharp/blob/13e74fea8dcda826d602d1ee534b67f46e93bd15/CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs#L165 

https://github.com/cefsharp/CefSharp/wiki/General-Usage#loading-htmlcssjavascriptetc-from-diskdatabaseembedded-resourcestream has examples of loading html from disk/in memory etc.

**Reply by AleidaBOX** [reply]
Thanks, I've been able to incorporate it into my form, I'm not so experienced in HTML though so I'll need to figure that out.
